actioncable/app/javascript/action_cable/connection.js

```diff
@@ -151,6 +151,7 @@ Connection.prototype.events = {
       case message_types.rejection:
         return this.subscriptions.reject(identifier)
       default:
+        this.monitor.recordPing()
```

Comment:
I think, it makes sense to record any incoming message, including subscription confirmations and rejections; so, we can move this call up before the `switch` statement.

Reply:
@palkan Thanks, record ping was moved before the `switch` statement. I would appreciate if you review again